plugins/modules/hmc_user.py

```diff
@@ -737,7 +743,15 @@ def ensure_update(module, params):
                     filter_d['NAMES'] = attributes['new_name']
                 changed = True
 
+        if m_config["NAME"] == 'hscroot' and m_config["passwd"] is not None and hmc_user == 'hscroot':
```

Comment:
define a variable and assign hscroot string to it and use that variable wherever you are hardcoding

Reply:
Done